src/xref2/ref_tools.ml

```diff
@@ -806,10 +851,30 @@ let resolve_reference_dot env parent name =
   | (`C _ | `CT _) as p -> resolve_reference_dot_class env p name
   | `P _ as page -> resolve_reference_dot_page env page name
 
+let resolve_page_reference env (r : Reference.Page.t) =
+  match r with
+  | `Resolved _ -> failwith "unimplemented"
```

Comment:
The `` `Resolved `` constructor for pages is never constructed so it could be dropped from the type.